css/style.css

```diff
@@ -18,20 +21,23 @@ body {
   width: 100%;
   height: 750px;
   background-color: #ffff;
-  z-index: -111;
-  border-radius: 0px 0 0 100px;
+  z-index: -11;
+  border-radius: 0 0 0 100px;
+  background-image: url("/assest/header-shapes\ mobile@2x.svg");
+  background-repeat: no-repeat;
```

Comment:
- [ ] Please, ensure that you provide the right image path, so your background will be displayed as requested. if you still struggling, you can this [Link for help.](https://developer.mozilla.org/fr/docs/Web/CSS/background-image)
- [ ] hint: Be sure that the image name doesn't contain any empty space.

1. **You design** 
![image](https://user-images.githubusercontent.com/80612925/170344989-a90e1982-0def-482e-92e8-8def1aa2eb0d.png)

2.**Figma design**
![image](https://user-images.githubusercontent.com/80612925/170345043-94ab7a3f-0794-41b8-b63d-cc9f92f17ce4.png)